hub/powertoys/keyboard-manager.md

```diff
@@ -167,7 +167,7 @@ Currently Mac and Linux profiles are not included.
 
 ### Will this work on video games?
 
-It depends on how the game accesses your keys. Certain keyboard APIs do not work with Keyboard Manager.
+We do not suggest using Keyboard Manager when playing games as it may affect the game's performance. It will also depend on how the game accesses your keys. Certain keyboard APIs do not work with Keyboard Manager.
```

Comment:
Nit/semantics; difference between "We do not suggest using..." and "We suggest not to use..."
(I think the latter is better)

Reply:
Reworded. Thanks!